contracts/DrawBeacon.sol

```diff
@@ -191,7 +191,7 @@ contract DrawBeacon is IDrawBeacon, Ownable {
     }
 
     /// @inheritdoc IDrawBeacon
-    function calculateNextBeaconPeriodStartTime(uint256 _currentTime)
+    function calculateNextBeaconPeriodStartTime(uint256 __currentTime)
```

Comment:
Is this to avoid shadowing? I've renamed to `_time`: https://github.com/pooltogether/v4-core/blob/b039133da5ee0fb01cc2317620745646d428cbb4/contracts/DrawBeacon.sol#L194
So you probably need to rebase master.